openlibrary/templates/barcodescanner.html

```diff
@@ -5,6 +5,10 @@
 $ ctx.setdefault('cssfile', 'barcodescanner')
 
 <div id="page-barcodescanner">
+    <div class="scan">
+        <input type="checkbox" name="multiple-scan" id="multiple-scan"/>
+        <label for="multiple-scan">Scan Many</label> 
```

Comment:
```suggestion
        <label for="multiple-scan">$_('Scan Many')</label> 
```
Use i18n string here.